frontend/styles/globals.css

```diff
@@ -6,8 +6,7 @@ html,
 body {
   padding: 0;
   margin: 0;
-  font-family: InterVariable, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu,
-    Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  font-family: Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
```

Comment:
@pascalalferink is this something you can answer?

Reply:
No, I can't answer the question. 

I asked the UX-designer Kasper about this: He answered he designed with the font Roboto, In the future this will probably change, but for the MVP this is oke.

Reply:
I'm pretty sure Roboto won't be available on any machine (except perhaps a Chromebook?) unless we [add it as a web font](https://www.npmjs.com/package/@fontsource/roboto).